package.json

```diff
@@ -23,22 +23,25 @@
     "README.md",
     "LICENSE.txt",
     "CHANGELOG.md",
+    "index.d.ts",
     "parser.js",
     "UserAgentBag.js",
     "UserAgentBag.js.flow"
   ],
   "scripts": {
-    "pretest": "npm run lint && npm run build && npm run test:flow",
+    "pretest": "npm run lint && npm run build && npm run test:flow && npm run test:typescript",
     "build": "pegjs --output parser.js parser.pegjs",
     "lint": "standard --fix",
     "test": "ava",
-    "test:flow": "flow status"
+    "test:flow": "flow status",
+    "test:typescript": "tsc --strict test/UserAgentBag-types.test.ts"
```

Comment:
This doesn't actually run the tests, right? Is that desirable?

Reply:
Good point - renamed to `transpile:typescript` instead to be more clear.